src/LeagueToolkit.Meta.Dump/LeagueToolkit.Meta.Dump.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="../CommonNuget.props" />
+
+  <!-- information about the assembly/project/package -->
+  <PropertyGroup>
+    <Version>4.0.0-beta.8</Version>
```

Comment:
You can remove the version tag, the new CI pipeline will now automagically assign the same version to each published package